cli/tests/integration/eval_tests.rs

```diff
@@ -20,6 +20,30 @@ fn eval_p() {
   assert_eq!("3", stdout_str);
 }
 
+// Make sure that snapshot flags don't affect runtime.
+#[test]
+fn eval_randomness() {
```

Comment:
Can we have a test that checks that snapshots are actually predicatable?

Reply:
I'd be happy to hard code the snapshot checksum in the tests 0205db97c85b0a06d5b0ea6b23258e6f

We just have to have a comment on how to update it easily

Reply:
> I'd be happy to hard code the snapshot checksum in the tests (0205db97c85b0a06d5b0ea6b23258e6f)
> 
> We just have to have a comment on how to update it easily

The snapshot/checksum will change pretty much all the time (slightest JS change in this repo) so that would be impractical IMO

Reply:
Probably not that often -  only if you change the JS code. Many PRs would not update it. 

Maybe the hick-up isn't worth it. I feel like we might discover something if we watched the checksum change.